app/perfsymboltable.cpp

```diff
@@ -730,9 +732,16 @@ static PerfAddressCache::SymbolCache cacheSymbols(Dwfl_Module *module, quint64 e
     for (int i = 0; i < numSymbols; ++i) {
         GElf_Sym sym;
         GElf_Addr symAddr;
-        const auto symbol = dwfl_module_getsym_info(module, i, &sym, &symAddr, nullptr, nullptr, nullptr);
-        if (symbol)
-            cache.append({symAddr - elfStart, sym.st_size, symbol});
+        Elf *elfp;
+        GElf_Word shndx;
+        quint64 start = 0;
+        quint64 size = 0;
```

Comment:
remove

Reply:
It is used later - in commit 3414df5. It is just side-effect of split.